src/ontology/mondo-edit.obo

```diff
@@ -38496,27 +38496,42 @@ is_a: MONDO:0020115 {source="MONDO:Redundant", source="NCIT:C35434"} ! secondary
 
 [Term]
 id: MONDO:0002441
-name: Jervell-Lange Nielsen syndrome
+name: Jervell and Lange-Nielsen syndrome
 def: "An autosomal recessive inherited syndrome caused by mutations in the KCNE1 and KCNQ1 genes. It is characterized by congenital hearing loss and arrhythmia. It is a form of long QT syndrome." [NCIT:P378]
 subset: gard_rare
+subset: ordo_clinical_subtype {source="Orphanet:90647"}
+subset: prototype_pattern
 synonym: "Cardioauditory syndrome of Jervell and Lange-Nielsen" RELATED [GARD:0003048]
 synonym: "deafness, congenital, and functional heart disease" RELATED [GARD:0003048]
+synonym: "Jervell and Lange Nielsen syndrome" EXACT [NCIT:C84793]
+synonym: "Jervell and Lange-Nielsen syndrome 1" RELATED [MONDO:Lexical, OMIM:220400]
+synonym: "Jervell and Lange-Nielsen syndrome type 1" RELATED [MONDORULE:1, OMIM:220400]
 synonym: "Jervell and Lange-Nielson syndrome" EXACT [DOID:2842]
 synonym: "Jervell Lange-Nielsen syndrome" RELATED [GARD:0003048]
```

Comment:
could you make this exact? Chris automatically brought in a lot of synonyms and labeled them as related, when they really should be exact. thanks!

Reply:
@nicolevasilevsky I don't think these should be exact synonyms. 
These refer to the type 1 version of the disease. This term refers to the grouping term (for both type 1 and type 2)

Reply:
You are right! I misread it. Thanks Sabrina :)